### PR TITLE
Update PROJ to v7.0.0

### PR DESCRIPTION
**Pull request by urschrei**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/proj.rb [hunk 1]
```diff
@@ -17,6 +17,7 @@ class Proj < Formula
     depends_on "libtool" => :build
   end
 
+  depends_on "libtiff" => :build
```
Comment (Bo98) [inline Formula/proj.rb]: Judging by the linkage errors, this looks like it's a runtime dependency - not just a build one.

Comment (urschrei) [reply Formula/proj.rb]: Hmm. Removing the "build" directive doesn't seem to have worked.

**Comment by Bo98** [comment]
Ok, that's the PROJ test passing now. However, there are other failures due to the fact that this update is binary incompatible with existing builds. You will therefore need to revision bump:

* gdal
* libgaiagraphics
* libgeotiff
* liblwgeom
* librasterlite
* libspatialite
* mapserver
* osm2pgsql
* postgis
* spatialite-gui
* spatialite-tools

to recompile them with the new version of PROJ.

(One per commit please - structure the commit message like `formula_name_goes_here: revision bump for proj`)

**Comment by urschrei** [comment]
@Bo98 Can I do this using `brew bump-formula-pr [formula] --message [as above]`? Do I also need to specify a new tag and revision for each formula?

**Comment by Bo98** [comment]
`brew bump-revision --message="for proj" <formula>`

**Comment by Bo98** [comment]
Thanks @urschrei. That worked.

**Pull request closed without merging**